internal/internal_command_state_machine.go

```diff
@@ -385,11 +387,15 @@ func (h *commandsHelper) newRequestCancelNexusOperationStateMachine(attributes *
 	}
 }
 
-func (h *commandsHelper) newTimerCommandStateMachine(attributes *commandpb.StartTimerCommandAttributes) *timerCommandStateMachine {
+func (h *commandsHelper) newTimerCommandStateMachine(
+	attributes *commandpb.StartTimerCommandAttributes,
+	summary *commonpb.Payload,
```

Comment:
imo this should just be `*UserMetadata`  directly. The state machine doesn't need to know or care about summary and later on if/when more is added this won't need to change.

Reply:
:+1: Agreed can change

Reply:
Changed